.travis.yml

```diff
@@ -1,4 +1,7 @@
-language: python
+sudo: required
```

Comment:
Unfortunately, it looks like it is: without `sudo: required`, TravisCI uses the container-based virtualization environments (https://docs.travis-ci.com/user/ci-environment/), which for some reason won't allow `pip install --user`s to work correctly: check out line 375 at https://travis-ci.org/gaurav/phylo2owl/builds/169361177#L375, where it appears to correctly install `setuptools>=18.5`, but then reports `Successfully installed setuptools-12.0.5`!

I might be able to download Jena to /tmp and use it from there, instead of `sudo`-ing a folder in `/opt` for it. Would that help? I don't think it makes a difference if we need to `sudo` for `pip install` anyway.